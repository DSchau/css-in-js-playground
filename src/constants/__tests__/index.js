import * as CONSTANTS from '../';

test('it contains expected exports', () => {
  expect(Object.keys(CONSTANTS)).toMatchSnapshot();
});
