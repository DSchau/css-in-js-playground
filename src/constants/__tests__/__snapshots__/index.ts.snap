// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it contains expected exports 1`] = `
Array [
  "SMALL_UP",
  "MEDIUM_UP",
  "LARGE_UP",
  "StyledComponents",
  "Glamorous",
  "Aphrodite",
  "ReactJSS",
  "JSS",
  "CXS",
  "Radium",
]
`;
